### DISCUSSION TITLE
CRSF relay with telemetry

### DISCUSSION THREAD

**Discussion by nflyer1** [ExpressLRS/ExpressLRS#1450] [Ideas]
My elrs relay setup is works very simply: opentx transmitter module --> relay receiver TX --> relay transmitter sport (single wire) --> receiver in model, provides the control link. However, I can only read the lq on my models via the OSD through the goggles, for which I can see 2 solutions:

1. If the relay receiver was set to receive telemetry from the relay transmitter in a passthrough configuration, I could read rssi, lq, etc. from the receiver in the model. In the present configuration, opentx reads either the telemetry from the relay receiver, or a garbled form of telemetry from both receivers. Not very useful as the relay receiver is in a fixed position close by so relay lq is not a concern. Would it be possible to hook up a full TX-RX pair on the relay transmitter? On my namimno oled tx board I found a TX pad next to sport, hooked that up but didn't work as intended.

2. Could one reconfigure a backpack as a receiver? Then the backpack could function as a relay receiver and feed the control signal directly to the transmitter, creating an integrated solution for the relay use case.

**Comment by flacon2000** [comment]
Hello. Pls, what module are you using for "relay transmitter sport (single wire)"?

**Reply by yuriy-boyun** [reply]
Hi @nflyer1. Does relay works without telemetry? because for me it doesn't work in any way :(

**Reply by Enzaii** [reply]
Since elrs 3.3 you just have to bridge TX and RX with a 1k resistor on the relay receiver and wire RX to relay Transmitter S.port :)

**Reply by nflyer1** [reply]
> Hello. Pls, what module are you using for "relay transmitter sport (single wire)"?

I've used various tx modules, any module will work but I suggest one with a screen so you can configure it directly. There has been talk about how difficult it would be to set up a lua script approach for the relay tx, if there's a screen it doesn't matter. So betafpv, nanimno, radiomaster modules with a screen and joystick all work. Still no telemetry passthrough even with newer versions of elrs though so you only get relay rx lq, which is ofcourse not what you want

**Reply by nflyer1** [reply]
> Since elrs 3.3 you just have to bridge TX and RX with a 1k resistor on the relay receiver and wire RX to relay Transmitter S.port :)

Tried that but you still get lq from the relay receiver so not what we really need. Setting up a backpack as a relay rx might be the answer ultimately, as you get full tx-rx pair but there has to be some fiddling with code to make that happen so we'll have to see

**Reply by nflyer1** [reply]
> Hi @nflyer1. Does relay works without telemetry? because for me it doesn't work in any way :(

Yes, I've been using a crsf relay setup since the early days of xfire. Just connect relay receiver RXpad  to the relay tx sport pin. With elrs make sure to set the relay rx baudrate to 400000 or it ain't gonna work. Radio telemetry doesn't get relayed though so you will need to get lq and rssi dbm through your video feed, but you will get gps coordinates and other fc telemetry down to your handset, which is great.

**Reply by nflyer1** [reply]
> Hello. Pls, what module are you using for "relay transmitter sport (single wire)"?

I've used nanimno, betafpv and radiomaster modules with screens. All work fine and are set up in the same way.

**Comment by evgeny17387new** [comment]
@nflyer1 i am also trying to build one, once all connected, should i bind rc remote controller with relay receiver and bind relay tx module with aircraft ?

**Reply by nflyer1** [reply]
Yes, I suggest just use different binding phrases for handset/relayrx and relaytx/aircraft. Works seamlessly.

**Comment by dfebrihardy6** [comment]
@nflyer1 It didn't work. I've tried all the steps, using either the RX pad or the TX pad on the receiver, but the ELRS transmitter still won't connect to the drone. "no handset" is displayed on the transmitter's screen, I'm using EdgeTX. Is it possible?

![3](https://github.com/user-attachments/assets/e3f03782-1ae3-4ff1-87a1-7efeaad61206)
![2](https://github.com/user-attachments/assets/73295b14-3141-4b12-b0a9-b5e2baaa872a)

i've tried activate the "Use as AirPort Serial device" on the ELRS browser configurator in receiver side but still not working.

**Reply by nflyer1** [reply]
Don't use airport. Reread the above instructions and it should work even without the resister, unless you've got a hardware issue

**Reply by dfebrihardy6** [reply]
> Don't use airport. Reread the above instructions and it should work even without the resister, unless you've got a hardware issue

wow it works now, thanks @nflyer1
